beacon-chain/flags/base.go

```diff
@@ -152,4 +152,14 @@ var (
 			"If such a sync is not possible, the node will treat it a critical and irrecoverable failure",
 		Value: "",
 	}
+	// EnableBackupWebhookFlag for users to trigger db backups via an HTTP webhook.
+	EnableBackupWebhookFlag = &cli.BoolFlag{
+		Name:  "enable-db-backup-webhook",
+		Usage: "Serve HTTP handler to initiate database backups. The handler is served on the monitoring port at path /db/backup.",
+	}
```

Comment:
Since doing backups is a good practice, shouldn't it be enabled by default?

Reply:
Not necessarily because some people might not want to expose that endpoint. Some might be wary of exposing an endpoint like that which can place a burden on a node if someone tries to spam it